library_management_system_downloader/tests_evaluate_api_response_for_fulltext_indication.py

```diff
@@ -0,0 +1,11 @@
+import evaluate_api_response_for_fulltext_indication as to_test
+
+
+def test_fulltext_indication():
+    assert to_test.fulltext_indication(
+            '<key id="full_text_indicator">true</key>') == 1
+
+    assert to_test.fulltext_indication('tester') == 0
+
+# Run the tests:
+test_fulltext_indication()
```

Comment:
This line not needed. pytest will run the function automatically because it starts with `test_`!

Not sure if `tests_evaluate_api_response_for_fulltext_indication.py` has to be renamed to `test_evaluate_api_response_for_fulltext_indication.py` for test detection.